koku/masu/api/expired_data.py

```diff
@@ -36,3 +36,23 @@ def expired_data(request):
     if simulate:
         response_key = response_key + " (simulated)"
     return Response({response_key: str(async_delete_results)})
+
+
+@never_cache
+@api_view(http_method_names=["GET", "DELETE"])
+@permission_classes((AllowAny,))
+@renderer_classes(tuple(api_settings.DEFAULT_RENDERER_CLASSES))
+def expired_trino_partitions(request):
+    """Return expired data."""
+    simulate = True
+    if request.method == "DELETE" and Config.DEBUG:
+        simulate = False
```

Comment:
Is this intentional? This endpoint can't be used in production.

Reply:
Yes, I wanted to keep the logic consistent with what we do for `expired_data` endpoint.

Reply:
sounds good!